### PR TITLE
 check_dns_sshfp.py: Expect empty answer

### PR DESCRIPTION
**Pull request by hasegeli**

### CODE REVIEW

File: src/check_dns_sshfp.py [hunk 1]
```diff
@@ -36,9 +37,10 @@
 
 def main():
     fqdn = getfqdn()
-    result = dns_query(fqdn, SSHFP, raise_on_no_answer=False)
 
-    if not result.rrset:
+    try:
+        result = dns_query(fqdn + '.', SSHFP)
```
Comment (TheFRedFox) [inline src/check_dns_sshfp.py]: What's about that extra dot?

File: src/check_dns_sshfp.py [hunk 2]
```diff
@@ -36,9 +37,10 @@
 
 def main():
     fqdn = getfqdn()
-    result = dns_query(fqdn, SSHFP, raise_on_no_answer=False)
 
-    if not result.rrset:
```
Comment (TheFRedFox) [inline src/check_dns_sshfp.py]: No need to check `result.rrset` additionally?

File: src/check_dns_sshfp.py (hunk 1 above)
Comment (hasegeli) [reply src/check_dns_sshfp.py]: I added a comment about it.

File: src/check_dns_sshfp.py (hunk 2 above)
Comment (hasegeli) [reply src/check_dns_sshfp.py]: No, we don't need because it would raise an exception without `raise_on_no_answer`.

Comment (TheFRedFox) [reply src/check_dns_sshfp.py]: Alright. 👍

**Pull request merged**